### ISSUE TITLE
HTTPS support

### ISSUE THREAD

**Issue by Montellese** [Montellese/mediaimporter.emby#9]
Right now the add-on only supports connecting to an Emby server using HTTP. HTTPS is not supported. It must also be possible to bypass HTTPS server certification validation by appending `|verifypeer=false` to URLs passed to Kodi.

- [ ] How can the add-on determine that it should use HTTPS?
- [ ] Make certificate verification configurable / optional.
  - [ ] Tell `requests` and `websocket` to ignore self-signed certificates.
  - [ ] Append `|verifypeer=false` to every URL directly used by Kodi.

**Comment by Montellese** [comment]
* If the user manually adds the media provider and provides an HTTPS URL this seems to work in general.
* If the media provider is auto-detected it always uses HTTP. How can the add-on determine that it should use HTTPS?
* If the media provider uses a self-signed HTTPS certificate all requests directly from Kodi will fail and require `|verifypeer=false` in the URL.

**Comment by darthShadow** [comment]
Would it be possible to also consider HTTP/2 support as a part of this issue? Plugging in the `hyper` library into requests for HTTP/2 support is only a single-line addition and should be backwards compatible.

Documentation: https://hyper.readthedocs.io/en/latest/quickstart.html#requests-integration

I can create a separate issue for this if you would prefer that.

Thanks.

**Comment by Montellese** [comment]
Yes please create a separate issue. Do you happen to know whether `hyper` is available as a module in Kodi's python environment?

**Comment by darthShadow** [comment]
Done. #33 

> Do you happen to know whether hyper is available as a module in Kodi's python environment?
Sorry, not sure of that.
